### DISCUSSION TITLE
企业微信自建应用通知配置，提交审批流，审批人在企业微信中接收不到应用通知消息？

### DISCUSSION THREAD

**Discussion by zhu547043537** [hhyo/Archery#1828] [Q&A] [answered]
archery1.8.5版本中，配置了CorpId 、AgentId、 Secret，对应的人员信息中企业微信UserId也配置了。但是在提交审批流，审批人在企业微信中接收不到应用通知消息？这个哪位大神遇到过，帮忙指点下

**Comment by nick2wang** [comment, marked as answer]
我这边可以验证企业微信应用通知在archery中是可以跑通的，所以你可以先测试下微信获取token和发消息两个接口，等调试可用了再去archery配置

**Reply by zhu547043537** [reply]
谢谢提示，自己写了个java的demo测试了下，是通的，结果发现是企业微信自建的应用，企业可信IP没有配置，配置了就可以了，感谢
